src/utils/statusCodes.js

```diff
@@ -0,0 +1,14 @@
+const statusCodes = {
+  OK: 200,
```

Comment:
```suggestion
const STATUS_CODES = {
  ok: 200,
```
Should be vise versa, constant name in UPPER_CASE and keys in camelCase